src/engine/ruleset/decoders/sonicwall/firewall.yml

```diff
@@ -0,0 +1,66 @@
+---
+name: decoder/sonicwall-firewall/0
+
+metadata:
+  module: Firewall
+  title: Sonicwall-Firewall logs decoder
+  description: Decoder for Firmware module logs belonging to Sonicwall
+  version: [6.5]
+  author:
+    name: Wazuh Inc. info@wazuh.com
+    date: 2023-01-12
+  references:
+    - https://www.sonicwall.com/es-mx/support/technical-documentation
+
+sources:
+  - decoder/syslog/0
+  - decoder/queue-localfile/0
+
+check:
+  - message: +r_match/^id\=[[:alpha:]]*? sn\=[[:alnum:]]*? time\=.*? fw
+
+parse:
+  logpar:
+    # Jan  3 13:45:36 192.168.5.1 id=firewall sn=000SERIAL time="2007-01-03 14:48:06" fw=89.160.20.156 pri=6 c=262144 m=98 msg="Connection Opened" n=23419 src=2.2.2.2:36701:WAN dst=89.160.20.156:50000:WAN proto=tcp/50000
+    - message: <~tmp/kv/=/ /"/'>
+
+normalize:
+  - map:
+      - event.code: $~tmp.m
+      - event.dataset: sonicwall.firewall
+      - event.kind: event
+      - event.module: sonicwall
+      - fileset.name: firewall
+      - input.type: log
+      - observer.product: Firewalls
+      - observer.type: Firewall
+      - observer.vendor: Sonicwall
```

Comment:
Why are sometimes the words with the first letter Uppercase and in other cases lowercase? Is that right?

Reply:
It is like this to respect the json expected of elastic
![Captura desde 2023-02-01 12-25-06](https://user-images.githubusercontent.com/63238635/216085142-f77c240e-fefb-4b79-99d0-c54a2cbd5667.png)